### ISSUE TITLE
BUG: VSI Setup Boundary Condition

### ISSUE THREAD

**Issue by birnstiel** [idefix-code/idefix#262]
### Describe the issue:

I think the inner radial boundary of the VSI setup is not correct (I hope I'm not being stupid again). [This line](https://github.com/idefix-code/idefix/blob/2f15373cb58387d67a91bd1e289416aab4813657/test/HD/VSI/setup.cpp#L48):

```
Vc(VX3,k,j,i) = R*Omega;
```

defines Keplerian rotation for the ghost cells, but [just above](https://github.com/idefix-code/idefix/blob/2f15373cb58387d67a91bd1e289416aab4813657/test/HD/VSI/setup.cpp#L35) omega is initialized as `Omega = 1.0` and not changed anywhere else. I think `Omega` can be removed and the velocity could be set as:

```
Vc(VX3,k,j,i) = 1.0/sqrt(R);
```

### runtime information:

Issue is architecture independent.

**Comment by glesur** [comment]
Yes, well, there is no "perfect" boundary conditions for VSI. The one in test imposes a solid body rotation, so that there is no free energy to tap into in the ghost zones, but it creates artefacts.

The best configuration is to actually add a wave-killing zone. What you propose can help, but is not full-proof, unfortunately.

**Comment by birnstiel** [comment]
Ok I see. I now saw that the inner boundary is at R=1, hence this is not an issue unless the radial domain is changed.

Somewhat related question: how is the dust boundary condition defined? Does it always equal the gas boundary condition of that side & direction? I.e. if I set a boundary to _outflow_, it will be outflow in dust _and_ gas while if I set it to _userdef_ I need to define it for both dust and gas? So there is no _gas=outflow_ while _dust=userdef_?

**Comment by glesur** [comment]
We have discussed the issue of boundary conditions for gas+dust mixture during the last idefix days. As of now, the boundary conditions are common to every fluid, so if it's `outflow` for the gas, it's `outflow` for the dust. This was initially to be retro-compatible with pluto.

The plan is to change that, and define boundary conditions separately for each fluid. I'll therefore keep this issue opened until this is done.
